src/aerialmap_display.cpp

```diff
@@ -44,19 +46,21 @@ limitations under the License. */
 #include "General.h"
 
 int constexpr FRAME_CONVENTION_XYZ_ENU = 0;  //  X -> East, Y -> North
-int constexpr FRAME_CONVENTION_XYZ_NED = 1;  //  X -> North, Y -> East
-int constexpr FRAME_CONVENTION_XYZ_NWU = 2;  //  X -> North, Y -> West
 
 namespace rviz
 {
-AerialMapDisplay::AerialMapDisplay() : Display(), dirty_(false), received_msg_(false)
+AerialMapDisplay::AerialMapDisplay() : Display(), tfListener_{ tfBuffer_ }, dirty_(false), received_msg_(false)
 {
-  topic_property_ =
-      new RosTopicProperty("Topic", "", QString::fromStdString(ros::message_traits::datatype<sensor_msgs::NavSatFix>()),
-                           "nav_msgs::Odometry topic to subscribe to.", this, SLOT(updateTopic()));
+  topic_property_ = new RosTopicProperty(
+      "NavSatFix", "", QString::fromStdString(ros::message_traits::datatype<sensor_msgs::NavSatFix>()),
+      "nav_msgs::NavSatFix topic to subscribe to, this determines the link between geo-coordinates (lat/lon/alt) and "
+      "local Cartesian coordinates. The frame_id of this topic is assumed to be exactly at the lat/lon/alt coordinates "
+      "of this message.",
+      this, SLOT(updateTopic()));
 
-  frame_property_ = new TfFrameProperty("Robot frame", "world", "TF frame for the moving robot.", this, nullptr, false,
-                                        SLOT(updateFrame()), this);
+  frame_property_ = new TfFrameProperty("Robot frame", "robot", "TF frame for the moving robot, must be connected to "
```

Comment:
Due to backwards compatibility this name shouldn't be `Robot frame` and something different.

Reply:
Well this actually is the robot frame (for the first time), so why shouldn't it be named that way?